packages/graphql-codegen/src/graphqlUtils.ts

```diff
@@ -1,8 +1,7 @@
 import { DocumentNode, InputObjectTypeDefinitionNode, ObjectTypeDefinitionNode, parse, print, visit } from "graphql";
-import { mapSimpleDbTypeToTypescriptType } from "joist-codegen";
+import { PrimitiveTypescriptType } from "joist-codegen/build/EntityDbMetadata";
```

Comment:
Hm, does this need to be from the `/build/` directory? Usually just going to the top-level module is more kosher for cross-module imports.